doc/rst/language/spec/interoperability.rst

```diff
@@ -203,35 +203,23 @@ and external code.
 Referring to Standard C Types
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In Chapel code, all standard C types must be expressed in terms of their
-Chapel equivalents. This is true, whether the entity is exported,
-imported or private. Standard C types and their corresponding Chapel
-types are shown in the following table.
-
-=========== =========== ============ ============ ======== ===========
-C Type      Chapel Type C Type       Chapel Type  C Type   Chapel Type
-=========== =========== ============ ============ ======== ===========
-int8_t      int(8)      uint8_t      uint(8)      \_real32 real(32)
-int16_t     int(16)     uint16_t     uint(16)     \_real64 real(64)
-int32_t     int(32)     uint32_t     uint(32)     \_imag32 imag(32)
-int64_t     int(64)     uint64_t     uint(64)     \_imag64 imag(64)
-chpl_bool   bool        const char\* c_string             
-\_complex64 complex(64) \_complex128 complex(128)         
-=========== =========== ============ ============ ======== ===========
-
-Standard C types are built-in. Their Chapel equivalents do not have to
-be declared using the ``extern`` keyword.
-
-In C, the “colloquial” integer type names ``char``, ``signed char``,
-``unsigned char``, (``signed``) ``short`` (``int``), ``unsigned short``
-(``int``), (``signed``) ``int``, ``unsigned int``, (``signed``) ``long``
-(``int``), ``unsigned long`` (``int``), (``signed``) ``long long``
-(``int``) and ``unsigned long long`` (``int``) may have an
-implementation-defined width. [5]_. When referring to C types in a
-Chapel program, the burden of making sure the type sizes agree is on the
-user. A Chapel implementation must ensure that all of the C equivalents
-in the above table are defined and have the correct representation with
-respect to the corresponding Chapel type.
+When referring to standard C types within Chapel code, they must
+either be expressed:
+
+* using the type aliases provided by the :mod:`CTypes` module,
+
+* using an external type declaration (see the next section), or
+
+* in terms of their Chapel equivalents (e.g., C ``double`` corresponds
+  to Chapel ``real(64)``)
+
+  .. note::
+
+     Note that a challenge to this last approach is that certain C
+     types—particularly integers—can vary in size between
```

Comment:
can we have some spaces around the two dashes here?

Reply:
Note that these are m-dashes, so in rendered form, I think they look fine (as I understand it, spaces around m-dashes are considered discouraged/redundant except in newspaper forms due to the very narrow columns):
<img width="695" alt="Screen Shot 2022-03-16 at 10 42 32 AM" src="https://user-images.githubusercontent.com/7536222/158653966-c7c495b6-4ffc-40d7-94a4-ab0fc7ab86b4.png">

Let me know if you disagree with this choice.

Reply:
https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use seems to say it could go either way on the spacing. For reading it in a text editor I think the spaces would be nice. However I do not need to dig in my heels here and I'm pretty sure you care way more about dash variants than I do.

Reply:
Note that viewing the .rst file in a text editor that doesn't use the same character for m-dash, n-dash, and hyphen also looks fine.  But in any case, I think we should be optimizing for the rendered HTML form of the page rather than the fixed-width text editor view, and (personally) find spaces with m-dashes in a "normal-sized" column of text to feel distractingly spaced out (I agree that both forms are acceptable, but haven't found many sources that prefer the spaces outside of newspapers—and, I guess, magazines according to M-W?  Now I'm going to be looking at magazines more carefully).